services/processing-queue.ts

```diff
@@ -5,73 +5,69 @@ export class ProcessingQueue {
   toExecute: (args: any) => Promise<unknown> | null;
   queue: Array<Term>;
   manualQueue: Array<Term>;
-  executing: boolean;
-  queuePollInterval: number;
+  isExecuting: boolean;
 
   constructor() {
     this.queue = [];
     this.manualQueue = [];
     this.run();
-    this.executing = false;
-    this.queuePollInterval = 60000;
+    this.isExecuting = false;
   }
 
   async run() {
-    if (this.queue.length >= 1 && !this.executing) {
+    if (this.queue.length >= 1 && !this.isExecuting) {
       try {
-        this.executing = true;
+        this.isExecuting = true;
         const subject = this.queue?.shift();
         if (subject) {
+          console.log(`|> TASK start for ${subject.value}`);
           await this.toExecute(subject);
         }
-        console.log(`|> Remaining number of tasks ${this.queue.length}`);
+        console.log(`|> Remaining number of tasks ${this.queue.length} \n`);
       } catch (error) {
         console.error(`|> Error while processing delta in queue ${error}`);
       } finally {
-        this.executing = false;
-        console.log('|> Done with action for queue item');
+        this.isExecuting = false;
+        console.log('|> TASK done \n|>\n');
         this.run();
       }
-    } else {
-      setTimeout(() => {
-        console.log(
-          `|> Trigger run, ${this.queue.length} remaining queue items and ${this.manualQueue.length} manual queued items`,
-        );
-        this.run();
-      }, this.queuePollInterval);
```

Comment:
This is removed by a trigger by action. This also makes that it is not running the setTimeout when nothing is triggered in an hour, which is good :D 

Result log

```bash
mandataris-1  | 2024-07-03T14:45:01.505693488Z |> Trigger endpoint mandatees-decisions
mandataris-1  | 2024-07-03T14:45:01.505836945Z |> CURRENT QUEUE ITEMS 0 and 0 in manual queue
mandataris-1  | 2024-07-03T14:45:01.509154185Z |> Moving 240 items from manual queue to the acutal executing queue.
mandataris-1  | 2024-07-03T14:45:01.509196090Z |> [2024-07-03T14:45:01.505Z] Added 240 to queue.
mandataris-1  | 2024-07-03T14:45:01.509202969Z |> Currently 240 items in queue.
mandataris-1  | 2024-07-03T14:45:01.509208565Z |> Queue was not empty triggering run()
mandataris-1  | 2024-07-03T14:45:01.509228923Z |> TASK start for http://data.lblod.info/id/mandatarissen/2743c2dc-818d-42aa-b4c0-1e972a53e18f

```